src/components/Group/Group.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "../../tests/tests.utils"
+import { Group } from "./Group"
+
+describe("<Group /> component", () => {
+  it("render with proper text", () => {
+    render(<Group title="Wydziały" elements={[]} />)
```

Comment:
Jeżeli testujemy coś co może mieć elementy wewnętrzne `elements` to musimy przetestować czy te elementy poprawnie przyjmuje i renderuje.